src/main/java/com/backbase/ct/bbfuel/setup/CapabilitiesDataSetup.java

```diff
@@ -203,5 +206,14 @@ private void ingestAccountStatementForSelectedUser() {
             });
          }
      }
+
+    private void ingestPositivePayChecksForSelectedUser() {
+        if (this.globalProperties.getBoolean(PROPERTY_INGEST_POSITIVE_PAY_CHECKS)) {
+            String externalUserIds = this.globalProperties.getString(PROPERTY_POSITIVEPAY_USERS);
```

Comment:
instead of maintaining a set of users that can have positive pay checks, why don't you leverage the job profiles & assign a specific job profile with permissions to positive pay, and based on the user's job profile you ingest this data?